crates/autopilot/src/domain/fee/mod.rs

```diff
@@ -51,8 +51,8 @@ impl ProtocolFee {
                     &order.data.sell_amount,
                     &order.data.buy_amount,
                     &order.data.fee_amount,
-                    &quote.buy_amount,
                     &quote.sell_amount,
+                    &quote.buy_amount,
```

Comment:
Could have been caught easier with 2 structs with named fields like Felix suggested. Maybe it makes sense to make that change as well in this PR.

Reply:
added